.env_example

```diff
@@ -10,6 +10,4 @@ export APP_ENV=local
 
 export PORT=3000
 
-export SDR_USER=sdrUser
 export SDR_PASS=sdrPass
-
```

Comment:
Please leave this file unchanged.  If the changes in this PR are complete, this file might be entirely redundant or the changes elsewhere might allow ENV values to override others.